src/screens/Shared/ViewBranchMachine/components/TabPaymentsReceived/index.tsx

```diff
@@ -0,0 +1,246 @@
+import { Button, Col, Row, Table } from 'antd';
+import { ColumnsType } from 'antd/lib/table';
+import { RequestErrors, TableHeader, TimeRangeFilter } from 'components';
+import {
+	ViewTransactionModal,
+	formatDateTime,
+	getFullName,
+	useCollectionReceipts,
+	CollectionReceipt,
+	ViewCollectionReceiptModal,
+} from 'ejjy-global';
+import {
+	DEFAULT_PAGE,
+	DEFAULT_PAGE_SIZE,
+	pageSizeOptions,
+	refetchOptions,
+	timeRangeTypes,
+	transactionStatuses,
+} from 'global';
+import { useQueryParams, useSiteSettingsNew, useTransactions } from 'hooks';
+import _ from 'lodash';
+import React, { useEffect, useState } from 'react';
+import { TransactionsCancelled } from 'screens/Shared/Branches/components/TabTransactions/components/TransactionsCancelled';
+import { convertIntoArray, formatInPeso, getLocalApiUrl } from 'utils';
+import { Summary } from './components/Summary';
+
+const columns: ColumnsType = [
+	{ title: 'Datetime', dataIndex: 'datetime' },
+	{ title: 'Receipt Type', dataIndex: 'receipt_type' },
+	{ title: 'Reference Number', dataIndex: 'invoice' },
+	{ title: 'Payment', dataIndex: 'payment' },
+	{ title: 'Cashier', dataIndex: 'cashier' },
+	{ title: 'Mode of Payment', dataIndex: 'mode_of_payment' },
+	{ title: 'Remarks', dataIndex: 'remarks' },
+];
+
+const invoiceTypes = {
+	sales_invoice: 'Sales Invoice',
+	charge_invoice: 'Charge Invoice',
+};
+
+const modeOfPayment = {
+	cash: 'Cash',
+	credit_pay: 'Credit Card',
+	check: 'Check',
+	others: 'Others',
+};
```

Comment:
Create a method instead if you're only using this to display the description. Check the other getter methods in `/utils/ui.tsx` in `ejjy-global`.